server/build/webpack/base.config.js

```diff
@@ -0,0 +1,318 @@
+import path, {sep} from 'path'
+import fs from 'fs'
+import webpack from 'webpack'
+
+import nodeExternals from 'webpack-node-externals'
+import UglifyJSPlugin from 'uglifyjs-webpack-plugin'
+import CaseSensitivePathPlugin from 'case-sensitive-paths-webpack-plugin'
+import WriteFilePlugin from 'write-file-webpack-plugin'
+import FriendlyErrorsWebpackPlugin from 'friendly-errors-webpack-plugin'
+import ExtractTextPlugin from 'extract-text-webpack-plugin'
+import {getPages} from './utils'
+import CombineAssetsPlugin from '../plugins/combine-assets-plugin'
+import PagesPlugin from '../plugins/pages-plugin'
+import DynamicChunksPlugin from '../plugins/dynamic-chunks-plugin'
+import findBabelConfig from '../babel/find-config'
+
+const nextDir = path.join(__dirname, '..', '..', '..', '..')
+const nextNodeModulesDir = path.join(nextDir, 'node_modules')
+const nextPagesDir = path.join(nextDir, 'pages')
+const defaultPages = [
+  '_error.js',
+  '_document.js'
+]
+const interpolateNames = new Map(defaultPages.map((p) => {
+  return [path.join(nextPagesDir, p), `dist/bundles/pages/${p}`]
+}))
+
+function babelConfig (dir, isServer) {
+  const mainBabelOptions = {
+    cacheDirectory: Boolean(process.env.BABEL_DISABLE_CACHE),
```

Comment:
Can't we use `next.config.js`. That's our configuration base. We don't want to support another way to config Next.js.

Anyway, what do we really need this. If really needed, can't we do this via the custom webpack config?